client/src/contexts/index.jsx

```diff
@@ -14,6 +21,11 @@ import {
 } from 'firebase/auth';
 import { auth, db } from '../firebase';
 
+const ridesCollectionReference = collection(db, 'rides');
```

Comment:
This looks solid. Let's divide these functions up and connect them to the components in the morning. Then we can start testing